src/ReadVTK.jl

```diff
@@ -443,6 +459,54 @@ function get_data(data_array::VTKDataArray{T,N,<:FormatAppended}) where {T,N}
 end
 
 
+"""
+    get\\_data\\_reshaped(data_array::VTKDataArray; cell_data=false)
+
+Retrieve actual data from a `VTKDataArray` and reshapes them as 1D, 2D or 3D arrays, in case we deal with structured grids.
+Note that vectors or tensors will have their components stored in the first dimension of the array. 
+As there is no way to automatically tell from the VTK file format whether it is a tensor, the user has to reshape this accordingly.
+
+"""
+function get_data_reshaped(data_array::VTKDataArray{T,Ncomponents}; cell_data=false) where {T,Ncomponents}
+
+  data = get_data(data_array);
+
+  # Retrieve size of grid
+  N   = get_local_size(data_array.vtk_file.xml_file, data_array.vtk_file.file_type, cell_data)
+  
+  # reshape 
+  if Ncomponents==1
+    data_reshape = reshape(data, N...)
+  else
+    data_reshape = reshape(data, Ncomponents, N...)
+  end
+
+  return data_reshape
+ 
+end
+
+"""
+  get\\_local\\_size(xml_file, file_type, cell_data=false)
+
+Retrieve the local size of a structured grid (ImageData, RectilinearGrid). Note that this always returns 3 dimensions, even if the data is 1D or 2D. 
+"""
+function get_local_size(xml_file, file_type, cell_data=false)
+
+  root = LightXML.root(xml_file)
+  dataset_element = root[file_type][1]
+  whole_extent = parse.(Int, split(attribute(dataset_element, "WholeExtent", required=true), ' '))
+
+  N = whole_extent[2:2:end] - whole_extent[1:2:end-1] .+ 1
+
+  if cell_data
+    N = N .- 1
+    N[N.==0] .= 1
+  end
+
+
+  return N
```

Comment:
```suggestion
  Nlocal_size = whole_extent[2:2:end] - whole_extent[1:2:end-1] .+ 1

  if cell_data
    local_size = local_size .- 1
    local_size[local_size .== 0] .= 1
  end

  return local_size
```
We often use `N` as a type parameter, thus I'd suggest a more descriptive name. Also, we use capital names only for types or type parameters, not for variables.